### DISCUSSION TITLE
Reciprocity as a Business Model

### DISCUSSION THREAD

**Discussion by rjheby** [PullDating/Org#126] [Revenue and Funding]
Let's ask for people to donate after good outcomes, so we incentivize these instead of bad outcomes. Got a date? Buy our devs a coffee too! Found a partner, want to get us dinner?

https://briefthoughts.substack.com/p/reciprocity-as-a-business-model?s=r&utm_campaign=post&utm_medium=web&utm_source=direct

**Comment by WilliamDormer** [comment]
I like this idea. It is a good way to connect developer and user incentives. One issue I do see is that people that are successful on the app are most likely to leave it right after, and thus may not actually be in the situation where they see the prompt to donate.

**Reply by Oshyan** [reply]
We'll have their email or text contact info. I guess this might link to another idea to send a survey when deactivating people's accounts #124 to try to find out why they stopped using it. In that survey's "completion" screen or in a follow-up *if* they say "met someone", we could send a small, open invitation to donate if they found the platform valuable.
